packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -300,7 +301,9 @@ describe('backupNg', () => {
       expect(vmTask.data.id).toBe(vmIdWithoutDisks)
     })
 
-    it('fails trying to run backup job without retentions', async () => {
+    // Not handled by the new backup implementation.
+    // See https://github.com/vatesfr/xen-orchestra/commit/0811da90146647aa2c0894d39b4bc41e39859453
+    it.skip('fails trying to run backup job without retentions', async () => {
```

Comment:
If the current behavior does not cause any issues, I agree we can remove this test.